packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js

```diff
@@ -1,43 +1,42 @@
 import React from 'react';
 import { View } from 'react-native';
+import { compose } from 'recompose';
 import PropTypes from 'prop-types';
-import Color from 'color';
-import { styled } from '@apollosproject/ui-kit';
+import { styled, withTheme } from '@apollosproject/ui-kit';
 import TileImage from '../../ui/TileImage';
 
+const enhance = compose(withTheme());
+
 const Square = styled(({ theme, ...props }) => ({
```

Comment:
should this be called something besides "Square"?

Reply:
Haha ... yeah, probably. `Tile` okay?

Reply:
`Tile` and `TileImage` should prob be a little more clear as to what they are

Reply:
What do you have in mind? I mean, they are tiles and images for the tiles. `TileImage` is actually something we brought over from core for some customizing (that I can't remember now). I'm open to other naming.